examples/gcal_oo_or.ty

```diff
@@ -90,7 +91,16 @@ p.add(
         Learning rate for the lateral excitatory projection to V1."""),
 
     inh_lr=param.Number(default=0.3,bounds=(0.0,None),doc="""
-        Learning rate for the lateral inhibitory projection to V1."""))
+        Learning rate for the lateral inhibitory projection to V1."""),
+
+    gpu=param.Boolean(default=False,bounds=(0,1),doc="""
+        Sets whether the simulation should be run on a GPU"""))
+
+
+if p.gpu:
+    from topo.gpu.projection import GPUSparseCFProjection
+    from topo.gpu.sheet import compute_sparse_gpu_joint_norm_totals, GPUSettlingCFSheet
+
```

Comment:
Actually, I'm not sure how I should proceed. I can't really merge both the sparse and GPU sparse models (`topo/tests/gcal_sparse.ty` and `topo/tests/gcal_sparse_gpu.ty`) into a single model that takes the GPU on/off as an argument, because there would be no way to use it in testing - there is no way to supply an additional argument, when launching training scripts in `runtests.py`.

What I could do instead is I could create a new file in the examples folder that takes an argument of whether GPU should be on/off and then loads an appropriate model from the topo/tests (`gcal_sparse.ty` or `gcal_sparse_gpu.ty`).

Does this make sense?